frontend/src/pages/Register.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import { Form, Button, Layout, Col, Divider } from 'antd';
+import { Typography } from 'antd';
+
+import RegisterForm from '@/forms/RegisterForm';
+import AuthLayout from '@/layout/AuthLayout';
+import SideContent from '@/components/SideContent';
+
+import logo from '@/style/images/logo.png';
+
+const { Content } = Layout;
+const { Title } = Typography;
+
+const RegisterPage = () => {
+  const onFinish = () => {};
```

Comment:
Could this event handler be implemented in such a way that values entered by the user are stored in a db or something? I can see that there are validation rules, which is great.

Reply:
Cool, I will be watching this space 😄

Reply:
I believe [this](https://github.com/idurar/idurar-erp-crm/pull/222/files) PR is related to the registration back end business logic